packages/tools/package/src/commands/changelog/changelog.ts

```diff
@@ -0,0 +1,54 @@
+import path from 'path';
+
+import { CommandsCommand } from '@webex/cli-tools';
+
+import { Package } from '../../models';
+import { Yarn } from '../../utils';
+import type { PackageConfig } from '../../models';
+
+import CONSTANTS from './changelog.constants';
+import type { Options } from './changelog.types';
+import { createOrUpdateChangelog } from './changelog.utils';
+
+/**
+ * @public
+ */
+const changelog: CommandsCommand<Options> = {
+  /**
+   * Configuration Object for this changelog Command configuration.
+   */
+  config: CONSTANTS.CONFIG,
+
+  /**
+   * Handles the creation of changelog for packages based on the provided Options
+   * from a Commands class instance.
+   *
+   * @param options - Options provided from the CLI interface.
+   * @returns - Promise that resolves once the process is complete.
+   */
+  handler: async (options: Options) => {
+    if (!options.packages) {
+      return Promise.resolve({});
+    }
+    const rootDir = process.cwd();
+
+    const tag = options.tag?.split('/').pop();
+    return Yarn.list()
+      .then((packageDetails) => packageDetails.map(
```

Comment:
Since we have already declared the function as async, we can use `async-await` syntax

Reply:
Removed async-await. Trying to keep all the tooling consistent with the .then approach

Reply:
I don't think we need to maintain consistency with thenable syntax. We should use async-await everywhere now, it's more readable.